spec.html

```diff
@@ -33488,7 +33488,7 @@ <h1>Array.prototype.pop ( )</h1>
         <emu-alg>
           1. Let _O_ be ? ToObject(*this* value).
           1. Let _len_ be ? LengthOfArrayLike(_O_).
-          1. If _len_ is zero, then
+          1. If _len_ is ±0, then
```

Comment:
`LengthOfArrayLike` cannot return `-0`, so the test can be `is +0` (or `is 0` if you like).
(And similarly for 2 subsequent edits.)